finalPrep/modZeroHero.js

```diff
@@ -48,24 +77,81 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 //assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
 
+function assessSituation2 (dangerLevel, announcement, excuse) {
```

Comment:
Curious why this function is here, it looks like the same logic/functionality to `assessSituation`?